post_install.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+OS_NAME=$TRAVIS_OS_NAME
+if [ "$OS_NAME" = "" ]; then
+  if [ $(uname -s) = "Darwin" ]; then
+    export OS_NAME=osx
+  else
+	  export OS_NAME=linux
+  fi
+fi
+
+if which xsltproc >/dev/null; then
+  if which wkhtmltopdf >/dev/null; then
+    @echo "installed"
+    #exit
+  fi
+fi
+
+if [ "$OS_NAME" = "linux" ]; then
+  sudo apt-get update -y
+  sudo apt-get install -y libxml2 xsltproc wget xfonts-75dpi
+  xsltproc --version
+  cd /tmp
+  wget https://github.com/wkhtmltopdf/wkhtmltopdf/releases/download/0.12.5/wkhtmltox_0.12.5-1.bionic_amd64.deb
+  sudo dpkg -i wkhtmltox_0.12.5-1.bionic_amd64.deb
+  sudo apt -f install
+  wkhtmltopdf --version
+fi
+
+if [ "$OS_NAME" = "osx" ]; then
+  brew install pkg-config libxml2 libxslt
+  sudo ln -s /usr/local/opt/libxml2/include/libxml2/libxml /usr/local/include/libxml
+  xsltproc --version
+  brew install wkhtmltopdf@0.12.5
```

Comment:
I think we need `brew link wkhtmltopdf` after the `install` step. The CI got this error

```
./post_install.sh: line 35: wkhtmltopdf: command not found
```